### ISSUE TITLE
[KEYBOARD]: Browser Back button often takes multiple clicks to return to Facility Locator search results

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#2749]
@1Copenut commented on [Thu Nov 29 2018](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15414)

## Issue

The Facility Locator browser history sometimes requires users to press the Back button between 3 and 7 times to return from a facility detail view to the facilities search results view. This could be an issue to keyboard and / or screen reader users who use the Back button. There's not a lot of data around Back button usage, and none for screen readers, so treating this as a high priority item.

## Acceptance Criteria

As a keyboard user, I want the browser's built-in Back button to navigate to the Facility Locator search results view in one click from a facility detail view.

## Possibly Related PRs

@1Copenut previously worked on the breadcrumb to make it remember search results. This was a result of a conversation with @mnorthuis about the Back button, so possibly related.

* https://github.com/department-of-veterans-affairs/vets-website/pull/9046/files

## Environment

* MacOS Mojave latest
* Chrome latest
* https://staging.va.gov/facilities/

## Steps to Recreate

1. Hard refresh one or more times to clear any local caches. On Chrome + Mac I pressed `Shift + Command + R`.
2. Enter `https://staging.va.gov/facilities/` in browser
3. Enter `Lincoln, NE` as your search term in Search near field
4. Switch Search for dropdown to Community Cares
5. Press `Search`
6. Click on `Dr. Frederick A. Mausolf`, should be the first result
7. Verify the URL should be https://staging.va.gov/find-locations/provider/ccp_1871787234
8. Press the browser's Back button. Verify the URL has changed, but the search results are not loaded.

## URL Stack Trace

The following are the URL changes I observed after taking steps 1-8 above. The relevant address information has been`REDACTED` for privacy. Please reach out to @1Copenut if that's a necessary piece of information for troubleshooting and I will provide it.

* https://staging.va.gov/facilities/?zoomLevel=9&page=1&address=Lincoln,%20NE&facilityType=cc_provider&location=40.79925662005228,-96.67007446289064&context=Lincoln,%20Nebraska,%20United%20States
* https://staging.va.gov/facilities/?zoomLevel=9&page=1&address=Lincoln,%20NE&facilityType=cc_provider&location=40.8001,-96.6674&context=Lincoln,%20Nebraska,%20United%20States
* https://staging.va.gov/facilities/?zoomLevel=9&page=1&address=Lincoln,%20NE&facilityType=cc_provider
* https://staging.va.gov/facilities/?zoomLevel=4&page=1&address=REDACTED&location=40.919908,-97.083899&context=68434
* https://staging.va.gov/facilities/?zoomLevel=4&page=1&address=REDACTED&context=68434
* https://staging.va.gov/facilities/?zoomLevel=4&page=1&address=REDACTED&location=40.91351257612758,-97.16308593750001&context=68434
* https://staging.va.gov/facilities/?zoomLevel=4&page=1&address=REDACTED&location=40.9198592,-97.0842112&context=68434
* https://staging.va.gov/find-locations/

## WCAG or Vendor Guidance (optional)

* [Keyboard: Understanding SC 2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html)

---

@mmiddaugh commented on [Tue Oct 22 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15414#issuecomment-545137192)

@brangi , here is one to evaluate for level of effort and usability impact

---

@brangi commented on [Thu Oct 24 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15414#issuecomment-546018575)

@mmiddaugh - I could not reproduce the issue with the steps above

**Comment by 1Copenut** [comment]
@brangi This one may not be an issue any more. It didn't occur every time I loaded a page, but appeared often enough I logged it as a defect. Usually I had to hard refresh 15-20 times to get it to catch. I think it may have had to do with API data returns being slow on some requests or maybe rate limiting?

**Comment by mmiddaugh** [comment]
@brangi has not been able to reproduce this issue.
